webapp/sources/api-doc/paths/nodes/id.yml

```diff
@@ -102,6 +102,16 @@ delete:
   operationId: deleteNode
   parameters:
     - $ref: ../../components/parameters/node-id.yml
+    - name: "mode"
+        in: query
+        required: false
+        description: Deletion mode to use, either move to trash ('move', default) or erase ('erase')
+        schema:
```

Comment:
I guess there is a default for compatibility, it should be documented with:

```yaml
default: move
```

Reply:
corrected in PR #3347